### PR TITLE
working on dotnet docs (RK-4782)

### PR DESCRIPTION
**Pull request by BenTheKlein**

### CODE REVIEW

**Comment by BenTheKlein** [comment]
@MickaelAlliel what about iis? is there a nice way to pass arguments?

**Comment by BenTheKlein** [comment]
@ElDuderinos Finally - the final step :)
docs are ready ; still need to publish to NuGet in the right name ; and do a feature flag to add bp on .cs files

File: docs/dotnet-setup.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+```cs
+public static void Start()
+public static void Start(RookOptions opts)
+public static void StartWithExceptions(RookOptions opts)
+```
+
+All methods initialize the SDK in the background using the optional configuration in the `opts` argument.  
+The simpler `Start` will never impact the application's flow, writing a failure to the console.  
+The `StartWithExceptions` will throw on error, so make sure to wrap the invocation with an appropriate `try`/`catch` block.
```
Comment (ElDuderinos) [inline docs/dotnet-setup.md]: Need a step by step explanation on integration using the env vars. The current doc isn't self explanatory enough.
I'm guessing the user needs to set the env vars and then use the Start() call without the opts param?

File: docs/sdk-setup.md [hunk 2]
```diff
@@ -17,6 +17,9 @@ If you are encountering any difficulties with deploying Rookout, this is the pla
 <li class="nav-item">
 <a class="nav-link-page" id="jvm-tab" data-toggle="tab" href="#jvm" role="tab" aria-controls="jvm" aria-selected="false">JVM</a>
 </li>
+<li class="nav-item">
+<a class="nav-link-page" id="dotnet-tab" data-toggle="tab" href="#dotnet" role="tab" aria-controls="dotnet" aria-selected="false">.Net Framework</a>
+</li>
```
Comment (ElDuderinos) [inline docs/sdk-setup.md]: @MickaelAlliel Any ideas on adding feature flags to the docs system? :(

File: docs/setup-intro.md [hunk 3]
```diff
@@ -12,5 +12,6 @@ To get started, pick your programming language or check out our [deployment exam
 1. [Python](python-setup.md)
 2. [Node.js](node-setup.md)
 3. [JVM](jvm-setup.md)
+4. [.NET Framework](dotnet-setup.md)
```
Comment (ElDuderinos) [inline docs/setup-intro.md]: feature flag

File: website/sidebars.json [hunk 4]
```diff
@@ -9,6 +9,7 @@
       "python-setup",
       "node-setup",
       "jvm-setup",
+      "dotnet-setup",
```
Comment (ElDuderinos) [inline website/sidebars.json]: feature flag

**Pull request merged**